src/io/InputStream.php

```diff
@@ -4,9 +4,27 @@
 
 namespace dmazurek\JsonRpc\io;
 
+use dmazurek\JsonRpc\request\JsonRpcRequestAggregate;
+use dmazurek\JsonRpc\request\JsonRpcRequestBuilder;
+
 class InputStream implements JsonRpcInput
 {
-    public function readFromInput(): string
+    /**
+     * @var JsonRpcRequestBuilder
+     */
+    private $requestBuilder;
+
+    public function __construct(JsonRpcRequestBuilder $requestBuilder)
+    {
+        $this->requestBuilder = $requestBuilder;
+    }
+
+    public function getRequest(): JsonRpcRequestAggregate
+    {
+        return $this->requestBuilder->buildFromJson($this->readFromInput());
```

Comment:
I think that'd be better to not to inject `JsonRpcRequestBuilder` in each class. The line `$this->requestBuilder->buildFromJson([...])` is repeated in each `JsonRpcInput`

Reply:
It is, but it is what resulted as a scope of JsonRpcServer built using tdd. I will think of it in next iteration